package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@amaabca/sensitive-param-filter",
-  "version": "1.0.6",
+  "version": "1.1.0",
```

Comment:
This isn't a new feature - just a bug fix.

Reply:
Actually, you don't need to do this at all, since the deployed code hasn't changed, so there's no need to release anything.